docs/setup.md

```diff
@@ -0,0 +1,43 @@
+In order to use the react eslint plugin do the following...
+
+1. At the root of your project type <code>npm install --save-dev eslint-plugin-react</code>
+2. Check your package.json 'devDependencies' for the following entries (version numbers may vary, but it's important that **the eslint version matches the peerDependency version in eslint-plugin-react**)...
+<pre>
+  "devDependencies": {
+    "eslint": "^7.3.1",
+    "eslint-plugin-react": "^7.10.3"
+  }
+</pre>
```

Comment:
i think maybe it'd be better to use instructions like here: https://github.com/airbnb/javascript/blob/c5bee75b1b358a3749f1a6d38ee6fad73de28e29/packages/eslint-config-airbnb-base/README.md#eslint-config-airbnb-base-1 so it's a bit more automated.